index.html

```diff
@@ -211,7 +214,7 @@ <h4>
   <div class="row">
     <div class="col-8">
       <h2>About</h2>
-      <p>cloud-init is developed and released as free software under both the<a class="p-link--external" href="https://www.gnu.org/licenses/gpl-3.0.en.html">GPLv3 open source license</a>and <a class="p-link--external" href="https://www.apache.org/licenses/LICENSE-2.0">Apache License version 2.0</a>. It was originally designed for the <a class="p-link--external" href="https://www.ubuntu.com/">Ubuntu</a> distribution of Linux in Amazon EC2, but is now supported on many Linux and UNIX distributions in every major cloud.</p>
+      <p>cloud-init is developed and released as free software under both the<a class="p-link--external" href="https://www.gnu.org/licenses/gpl-3.0.en.html">GPLv3 open source license</a>and the <a class="p-link--external" href="https://www.apache.org/licenses/LICENSE-2.0">Apache License version 2.0</a>. It was originally designed for the <a class="p-link--external" href="https://www.ubuntu.com/">Ubuntu</a> distribution of Linux in Amazon EC2, but is now supported on many Linux and UNIX distributions in every major cloud.</p>
```

Comment:
You didn't introduce this, but this para needs a space after "free software under both the" and after "GPLv3 open source license"